match/docs/openapi/lookup/id.yaml

```
@@ -4,6 +4,12 @@ get:
     - "Lookup"
   summary: "Get the original match data related to a Lookup ID"
   description: "User can provide a Lookup ID and receive the match data associated with it"
+  parameters:
+    - in: header
+      name: From
+      schema:
+        type: string
+      description: "The username of the individual making the request"
```

```suggestion
      description: "As in the HTTP/1.1 RFC, used for logging purposes as a means for identifying the source of invalid or unwanted requests. The interpretation of this field is that the request is being performed on behalf of the state government-affiliated person whose email address (or username) is specified here. It is not used for authentication or authorization."
```
We should clarify some more; I've copied and modified the `From` description from the HTTP RFC in the suggestion above.